CMakeLists.txt

```diff
@@ -95,6 +106,32 @@ qsk_setup_install()
 add_subdirectory(src)
 add_subdirectory(skins)
 
+include(CMakePackageConfigHelpers)
+
+# write QSkinnyConfigVersion.cmake file for cmake import
+write_basic_package_version_file(
+    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}ConfigVersion.cmake
```

Comment:
@lgtmak @uwerat I think we should replace the `CMAKE_BINARY_DIR` usages entirely with `CMAKE_CURRENT_BINARY_DIR` for the "QSkinny as a subdirectory" scenarion in #348

Thereby we don't need this `_QSkinny` subdirectory